internal/daysteps/daysteps.go

```diff
@@ -1,23 +1,62 @@
 package daysteps
 
 import (
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
 )
 
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
+// parsePackage разбирает строку с данными о шагах и времени.
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	parts := strings.Split(data, ",")
+
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("invalid data format: %s", data)
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("invalid steps format: %s", parts[0])
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```